lib/node_modules/@stdlib/array/complex64/README.md

```diff
@@ -1607,6 +1607,46 @@ var count = context.count;
 // returns 2
 ```
 
+<a name="method-subarray"></a>
+
+#### Complex64Array.prototype.subarray( \[begin\[, end]] )
+
+Creates a new typed array view over the same underlying [`ArrayBuffer`][@stdlib/array/buffer] and with the same underlying data type as the host array.
+
+```javascript
+var arr = new Complex64Array( [ 1.0, 2.0, 3.0, 4.0 ] );
+
+var subarr = arr.subarray();
+// returns <Complex64Array>
+
+var len = subarr.length;
+// returns 2
+```
+
+By default, the method creates a typed array view beginning with the first array element. To specify an alternative array index at which to begin, provide a `begin` index (inclusive).
+
+```javascript
+var arr = new Complex64Array( [ 1.0, 2.0, 3.0, 4.0 ] );
+
+var subarr = arr.subarray( 1 );
+// returns <Complex64Array>
+
+var len = subarr.length;
+// returns 1
+```
+
+By default, the method creates a typed array view which includes all array elements after `begin`. To limit the number of array elements after `begin`, provide an `end` index (exclusive).
+
+```javascript
+var arr = new Complex64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+
+var subarr = arr.subarray( 0, -1 );
```

Comment:
Instead of `(0,-1)`, I'd made the original array length `4` and then do `(1,-1)` to return a view of the two middle elements.